superset/views/core.py

```diff
@@ -542,6 +542,37 @@ class SliceAddView(SliceModelView):  # noqa
 appbuilder.add_view_no_menu(SliceAddView)
 
 
+class TagModelView(SupersetModelView, DeleteMixin):
+    datamodel = SQLAInterface(models.Tag)
+
+    list_title = _('List Tag')
+    show_title = _('Show Tag')
+    add_title = _('Add Tag')
+    edit_title = _('Edit Tag')
+
+    list_columns = ['tag_name']
+    edit_columns = ['tag_name']
+
+    add_columns = edit_columns
+
+    label_columns = {
+        'tag_name': _('Tag Name'),
+    }
+
+    def pre_delete(self, obj):
+        pass
+
+
+appbuilder.add_view(
+    TagModelView,
+    'Tags',
+    label=__('Tags'),
```

Comment:
should this be two underscores?

Reply:
Yeah looks like they use dunder for the labels